### PR TITLE
explicitly include user data/metadata residency preference

### PR DESCRIPTION
**Pull request by ara4n**
Discoverability & Reachability doesn't cover the very important problem of ensuring that users should be able to opt out of interoperating their conversation data/metadata with a given provider.

### CODE REVIEW

**Comment by ara4n** [comment]
as discussed in Q&A in the BOF

**Comment by ekr** [comment]
I actually think this is over-specification in the charter and is a requirement that the WG can discuss.

**Comment by ara4n** [comment]
@ekr: what about changing it to 'data processing' as per @vbertola's suggestion in the BOF? it feels pretty irresponsible not to consider the users' preferences on whether their conversations should be allowed to interoperate or not.

**Comment by ekr** [comment]
This is a category error. Just because we are not encoding some restriction in the charter doesn't mean we're not going to consider it.

**Comment by ara4n** [comment]
as discussed in the BOF; going to close it.

**Comment by ekr** [comment]
I think those are in scope because they define the work items, whereas this describes *how* things happen.

**Comment by ara4n** [comment]
> I think those are in scope because they define the work items, whereas this describes how things happen.

work items here would be:
 * preferences over discoverability (do i let people find me via MIMI?)
 * preferences over reachability (do i let people contact via MIMI?)
 * preferences over data residency (do i let people bridge my conversations to given providers over MIMI?)

I'm genuinely confused on why data residency considerations would be any different from discoverability or reachability - i may need an ELI5...

**Comment by ekr** [comment]
The work items are protocols/mechanisms for introduction and discovery, not
preferences. The preferences are constraints on those designs.

**Comment by ara4n** [comment]
@ekr gotcha - so the concern here was "the charter of the WG needs to define protocols/mechanisms for introduction and discovery functionality. but not protocols/mechanisms for a permission model for where your data is exposed - as this is not a functionality in its own right, but an attribute of sending messages in general"?

Sorry for being slow on coming up to speed with the semantics...
